### PR TITLE
feat: Landing Page added

### PR DESCRIPTION
**Pull request by rdshinde**
- Added landing page
- Refactored z-indexing
- removed variables for z-indexes 

https://user-images.githubusercontent.com/67017632/154454903-d886d265-3d58-4f06-be03-63c0e277dbdd.mov

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -100,23 +104,57 @@ a .fa-store:hover {
...
+.new-arrivals .card:hover {
+  transform: scale(1.01);
+  box-shadow: var(--shadow-color) 0px 5px 15px;
+}
+.new-arrivals .card .card__footer {
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  padding: var(--scale-sm);
+}
```
Comment (raman1999) [inline style.css:152]: Use utility class for flex and can override accordingly then

**Review by raman1999 — APPROVED** [review]
Rest looks good

File: style.css (hunk 1 above)
Comment (rdshinde) [reply style.css:152]: I have an utility class for it. Here due to specificity I am over-riding it.

**Review by vivekmehto — APPROVED** [review]
looks good to me. good use of utility classes

**Pull request merged**